Makefile.core.mk

```diff
@@ -258,7 +261,8 @@ deploy-yaml: helm ## Outputs YAML manifests needed to deploy the controller
 .PHONY: deploy-openshift # TODO: remove this target and use deploy-olm instead (when we fix the internal registry TLS issues when using operator-sdk run bundle)
 deploy-openshift: helm ## Deploy controller to OpenShift via YAML manifests
 	$(info NAMESPACE: $(NAMESPACE))
-	$(MAKE) -s deploy-yaml-openshift | kubectl apply -f -
+	export HELM_TEMPL_DEF_FLAGS="--include-crds --set image='${IMAGE}'" && \
```

Comment:
can we split this out into a separate commit and fix it everywhere?

Reply:
Sure, I'll do that